themes/beaver/layouts/page/free-consultation.html

```diff
@@ -107,7 +107,7 @@ <h1 class="fl-heading">
                                                       <div class="pp-icon-wrapper animated none">
                                                          <div class="pp-infolist-icon">
                                                             <div class="pp-infolist-icon-inner">
-                                                               <img decoding="async" src="{{ absURL "wp-content/uploads/2022/10/tick-circle-white.svg" }}" alt="" role="presentation" />
+                                                               <img src="{{ (resources.Get "assets_dynamic/img/tick-circle-white.svg" | resources.Fingerprint).Permalink }}" alt="" width="24" height="24" role="presentation">

```

Comment:
do not use `assets_dynamic` all files are dynamic

Reply:
reverted to not mix this with extract image resizing